experiments/benchmarks/land.jl

```diff
@@ -206,8 +206,16 @@ function setup_prob(t0, tf, Δt; nelements = (101, 15))
             soil_params_mask,
             0,
         )
-    PAR_albedo = ClimaCore.Fields.zeros(surface_space) .+ FT(0.2)
-    NIR_albedo = ClimaCore.Fields.zeros(surface_space) .+ FT(0.2)
+    # clm_data is used for g1, vcmax, rooting, soil albedo,  and two_stream param maps
+    clm_artifact_path = ClimaLand.Artifacts.clm_data_folder_path(; context)
+
+    PAR_albedo_dry, NIR_albedo_dry, PAR_albedo_wet, NIR_albedo_wet =
+        Soil.create_soil_albedo_vars(
```

Comment:
I think we can take this one extra step and return direclty the clm data. This function could be `clm_soil_albedo_properties` and just take the `surface_space`.